src/app/data-planes/components/DataPlaneDetails.vue

```diff
@@ -4,9 +4,9 @@
       <h1 class="entity-heading">
         DPP:
 
-        <TextWithCopyButton :text="dataPlane.name">
+        <TextWithCopyButton :text="props.dataplaneOverview.name">
```

Comment:
nitpick: sometimes is `dataPlaneOverview` and sometimes `dataplaneOverview`

Reply:
I’ve slowly started to name things using the `dataplane` rather than `dataPlane` because the resource names from the API are `Dataplane` and `DataplaneOverview` even though the `p` should technically be capitalized. I only did this for things here that I actually needed to touch to avoid making the diff bigger than it needed to be.